README.md

```diff
@@ -21,20 +18,34 @@ npm i -g helm-viewer
 ## Usage
 
 ```bash
-# When you are in a chart helm folder
+# Run in a Helm chart folder
 helm-viewer
 
-# To target a specific path
-helm-viewer path/of/the/chart
+# Specify a specific path
+helm-viewer path/to/the/chart
 ```
 
-It will analyze the helm chart and generate a HTML report, automatically opened in your favorite browser.
+Running the command will analyze the Helm chart and generate an HTML report, which will be automatically opened in your default web browser.
 
+![Screenshot of the output](./.github/ressources/demo-1.png)
 
 # Development
 ## Generate the binary
 
-```
+To generate the binary for the helm-viewer project, we use the `just` command, which is a task runner tool. It helps streamline various development tasks and automates common actions.

```

Comment:
Do you think that describing the `just` project here is ok? I would have only linked the project